pkg/kotsadmconfig/validation/types.go

```diff
@@ -0,0 +1,24 @@
+package validation
+
+import (
+	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
+	"github.com/replicatedhq/kots/kotskinds/multitype"
+)
+
+type ConfigGroupValidationError struct {
```

Comment:
let's move these to a separate `types` package inside the `validation` package.

Reply:
This wouldn't be needed as the `validator` is the package from where all the dependent functions will be referred from and the structs are more generic. 
The chances of cyclic dependency is low as `validator` package only refers to generic  `kotsv1beta1` `util` packages

Reply:
Finally decided to move to a separate package under `/pkg/kotskinds/types`
So all the ItemType Const and other config related structs can re-use the pacakge